test/ejabberd_sm_SUITE.erl

```diff
@@ -521,3 +504,28 @@ try_to_reproduce_race_condition(Config) ->
                                    [SID, Other]),
             {error, reproduced}
     end.
+
+setup_sm(Config) ->
+    case proplists:get_value(backend, Config) of
+        ejabberd_sm_redis ->
+            set_meck({redis, [{pool_size, 3}, {worker_config, [{host, "192.168.99.100"}, {port, 6379}]}]}),
```

Comment:
Did you check or do you expect that redis on travis is available on `192.168.99.100`? How about local envs, are we sure this will always be `192.168.99.100` for everyone?

Reply:
:+1:

Reply:
ooops.. I'm so sorry, these are of course leftovers